### PR TITLE
Solved Issue #9334. Added unpin feature for rewards list in tasks.

### LINKED ISSUE #9334: pinned Rewards on Tasks page need on-hover effect to show the pin/unpin icon
In the Rewards column on the Tasks page, when you hover over any pinned Reward, the pin icon should become visible at the top of the Reward and you should then be able to click on it to unpin the item.

I.e., when you hover, currently you see this:

image

The desired effect is the same but with this icon added:

image

Clicking that icon will unpin the Reward and make it disappear from the Rewards column immediately.

### PR DESCRIPTION
**Pull request by klezt**
Fixes #9334 

### Changes
Added Unpin button the same way it is used featured items in market. Message pops up if reward cannot be unpinned.

UUID: efcfbb29-939d-437f-bebe-058bdef04997

### CODE REVIEW

File: website/client/components/tasks/column.vue [hunk 1]
```diff
@@ -58,11 +58,50 @@
...
+              @click.prevent.stop="togglePinned(ctx.item)"
+            )
+              span.svg-icon.inline.icon-12.color(v-html="icons.pin")
+
 </template>
 
 <style lang="scss" scoped>
   @import '~client/assets/scss/colors.scss';
 
+  .badge-svg {
```
Comment (paglias) [inline website/client/components/tasks/column.vue]: this css isn't already available somewhere else?

File: website/client/components/tasks/column.vue [hunk 2]
```diff
@@ -586,6 +627,15 @@ export default {
         this.selectedItemToBuy = null;
       }
     },
+    togglePinned (item) {
+      try {
+        if (!this.$store.dispatch('user:togglePinnedItem', {type: item.pinType, path: item.path})) {
+          this.$parent.$parent.text(this.$t('unpinnedItem', {item: item.text}));
```
Comment (paglias) [inline website/client/components/tasks/column.vue]: you can import the notifications mixin into this component directly

File: website/client/components/tasks/column.vue (hunk 1 above)
Comment (paglias) [reply website/client/components/tasks/column.vue]: cc @negue

Comment (klezt) [reply website/client/components/tasks/column.vue]: @paglias I tried to import them using "@import '/client/components/shops/market/index/badge-svg.css';", however the class is not recognized.

Comment (paglias) [reply website/client/components/tasks/column.vue]: `/client/components/shops/market/index/badge-svg.css';` doesn't exist as a file, if you meant the `.vue` file then the css is only available inside that .vue file. You should move it inside a .scss file in client/assets/scss and then you can use it everywhere

**Comment by klezt** [comment]
@paglias Sorry for the issues, hope it is alright now.

File: website/client/components/shops/market/index.vue [hunk 3]
```diff
@@ -234,6 +234,7 @@
 <style lang="scss">
   @import '~client/assets/scss/colors.scss';
   @import '~client/assets/scss/variables.scss';
+  @import '~client/assets/scss/pin.scss';
```
Comment (paglias) [inline website/client/components/shops/market/index.vue]: If you could remove this too it would be great, thanks :)

**Review by paglias — CHANGES_REQUESTED** [review]

**Review by paglias — APPROVED** [review]

**Comment by paglias** [comment]
@klezt hey, I've merged the PR but looks like your UUID is not the correct one, can you check it? So that I can award your contributor tier, thanks!

**Comment by klezt** [comment]
@paglias UUID: 016566f9-6c31-4a07-b4e3-541a7d9bb077

**Comment by paglias** [comment]
tier awarded :)

**Pull request merged**